src/pages/stay-secure.js

```diff
@@ -0,0 +1,177 @@
+var React = require('react');
+var Footer = require('../components/footer.js');
+var ContentContainer = require('../components/content-container.js');
+import {Accordion, Tab} from '../components/mozilla-accordion.js';
+import ga from 'react-ga';
+import { browserHistory } from 'react-router';
+import StickContainer from '../components/sticky-container.js';
+
+module.exports = React.createClass({
+  onButtonClick: function() {
+    window.location.href = '/stay-secure/thank-you/?action=user_updated';
```

Comment:
We're tracking results for this action in google analytics. We look for page views for thank-you and count action=user_updated